ui/pages/add-token/add-token.component.js

```diff
@@ -167,7 +165,7 @@ class AddToken extends Component {
       autoFilled: false,
     });
 
-    const addressIsValid = isValidAddress(customAddress);
+    const addressIsValid = isValidHexAddress(customAddress, false);
```

Comment:
This behavior is pre-existing, but it does seem strange to me that we'd show an "invalid address" error when the address is mixed-case but doesn't match our checksum. Particularly when we're not checksumming the addresses properly according to EIP-1191, as we're omitting the chainID. As a result, we'd be showing "invalid address" when an address was properly checksummed in accordance with EIP-1191.